src/RoomsSDKAdapter.js

```diff
@@ -126,12 +135,56 @@ export default class RoomsSDKAdapter extends RoomsAdapter {
       );
 
       // Convert to a multicast observable
-      this.getRoomObservables[ID] = getRoom$.pipe(
-        publishReplay(1),
-        refCount(),
-      );
+      this.getRoomObservables[ID] = getRoom$.pipe(publishReplay(1), refCount());
     }
 
     return this.getRoomObservables[ID];
   }
+
+  /**
+   * Returns an observable that emits current and future activities from the specified room.
+   *
+   * @param {string} id ID of room to get
+   * @returns {Observable.<Activity>} Observable stream that emits current and future activities from the specified room
+   */
+  getRoomActivities(id) {
+    logger.debug('ROOM', id, 'getRoomActivities()', ['called with', {id}]);
+
+    // if (!(ID in this.getRoomObservables)) {
```

Comment:
Remove commented out code

Reply:
Do we want this to follow the cache pattern also being done in `getRooms`?